web/middleware.py

```
@@ -141,3 +142,56 @@ def __call__(self, request):
             # Report to Sentry
             sentry_sdk.capture_exception(e)
             return self.get_response(request)
+
+
+class QuizSecurityMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request) -> HttpResponse:
```

Ruff (0.8.2)

Missing type annotation for function argument `request`

(ANN001)